### PR TITLE
Docs

### LINKED ISSUE #4: online documentation
You can copy the files from pytng or RotamerConvolve to get the simple standard layout and color scheme. (Perhaps we should have a repo just for the basic doc layout for easier cloning.)

It should appear as https://www.mdanalysis.org/pmda

### PR DESCRIPTION
**Pull request by kain88-de**
fix #4

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+]
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'pytng', u'pytng Documentation',
```
Comment (orbeckst) [inline docs/conf.py]: still has pytng...

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+# The name for this set of Sphinx documents.  If None, it defaults to
+# "<project> v<release> documentation".
+#html_title = None
+
+# A shorter title for the navigation bar.  Default is the same as html_title.
+#html_short_title = None
+
+# The name of an image file (relative to this directory) to place at the top
+# of the sidebar. --- use theme
+html_logo = "_static/logos/mdanalysis-logo-200x150.png"
```
Comment (orbeckst) [inline docs/conf.py]: replace with mda-logo.png ?

**Comment by orbeckst** [comment]
I'll add the full MDA styling to this branch.

**Comment by kain88-de** [comment]
OK just push to this branch then. Thanks for helping

**Comment by kain88-de** [comment]
@orbeckst just merge this if you setup the travis build. I think you know better how this can be done.

**Comment by orbeckst** [comment]
ok

**Pull request merged**